internal/api/impl/v1/project/service.go

```diff
@@ -37,28 +37,39 @@ import (
 
 type service struct {
 	project.Service
-	dao            project.DAO
-	folderDAO      folder.DAO
-	datasourceDAO  datasource.DAO
-	dashboardDAO   dashboard.DAO
-	roleDAO        role.DAO
-	roleBindingDAO rolebinding.DAO
-	secretDAO      secret.DAO
-	variableDAO    variable.DAO
-	authz          authorization.Authorization
+	dao                   project.DAO
+	folderDAO             folder.DAO
+	datasourceDAO         datasource.DAO
+	dashboardDAO          dashboard.DAO
+	roleDAO               role.DAO
+	roleBindingDAO        rolebinding.DAO
+	secretDAO             secret.DAO
+	variableDAO           variable.DAO
+	authz                 authorization.Authorization
+	isAuthorizationNative bool
```

Comment:
Make a IsNative() method in `` authorization.Authorization`` would simplify a bit